### ISSUE TITLE
Sort from 1 instead of 0

### ISSUE THREAD

**Issue by charleschege** [nushell/nushell#1235]
When using `ls` command, the list of files is numbered from `0` instead of `1`. Fix the list to start from 1

**Comment by jonathandturner** [comment]
@charleschege - there are some difficulties here with being 1-based, unfortunately. It's true that if this were a standalone product, like Excel, we have more flexibility in where we start numbering from. That may lead us to naturally pick 1 as the starting point.

Unfortunately, Nu has a full plugin mechanism that allows any language to extend Nu with additional functionality. Programming languages traditionally are 0-based, and going between 0-based and 1-based causes lots of issues and generally just makes the extension experience worse.

It may still be worth the trade-off, so I've opened https://github.com/nushell/nushell/issues/1243 to track ideas we need to test from user feedback. I've added this idea as something to explore there. If it's a big win, we can make the change and then help plugin authors make the shift.

Closing this as it's now an area of exploration in #1243
